controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -1707,8 +1707,37 @@ func (r *BareMetalHostReconciler) updateEventHandler(e event.UpdateEvent) bool {
 }
 
 // SetupWithManager registers the reconciler to be run by the manager
-func (r *BareMetalHostReconciler) SetupWithManager(mgr ctrl.Manager, preprovImgEnable bool) error {
+func (r *BareMetalHostReconciler) SetupWithManager(mgr ctrl.Manager, preprovImgEnable bool, options controller.Options) error {
 
+	maxConcurrentReconciles := 0
+	if options.MaxConcurrentReconciles > 0 {
+		ctrl.Log.Info(fmt.Sprintf("Controller Concurrency will be set to %d according to command line flag", options.MaxConcurrentReconciles))
+	} else {
+		mCR, err := getMaxConcurrentReconciles()
```

Comment:
This is ~never called since the default value is 10, so we're not preserving the previous behaviour.

Reply:
Default is set to 0 now.